WordPress/src/main/java/org/wordpress/android/GCMMessageService.java

```diff
@@ -167,13 +163,15 @@ private void buildAndShowNotificationFromNoteData(Bundle data) {
             Bundle noteBundle = sActiveNotificationsMap.get(id);
             if (noteBundle != null && noteBundle.getString(PUSH_ARG_NOTE_ID, "").equals(noteId)) {
```

Comment:
I wonder why `note_id` is empty here. Aren't we always setting it on the server side?